Pinta.Tools/Tools/TextTool.cs

```diff
@@ -975,7 +984,35 @@ private void RedrawText (bool showCursor, bool useTextLayer)
 			ClearTextLayer ();
 		}
 
+		/*		var antialiasing = antialiasing_button.SelectedItem.GetTagOrDefault (true);
+		   if (antialiasing) {
+
+		   } else {
+		   	Settings.PutSetting (ANTIALIAS_SETTING, 1);
+		   	var options = new Cairo.FontOptions ();
+		   	options.Antialias = Antialias.None;
+		   	PangoCairo.Functions.ContextSetFontOptions (PintaCore.Chrome.MainWindow.GetPangoContext (), options);
+		   }
+*/
+
 		Cairo.Context g = new (surf);
+
+		if (UseAntialiasing) {
+			// Adjusts antialiasing JUST for the outline brush
+			g.Antialias = Cairo.Antialias.Gray;
+			// Adjusts antialiasing for PangoCairo's text draw function
+			var options = new Cairo.FontOptions ();
+			options.Antialias = Antialias.Gray;
+			PangoCairo.Functions.ContextSetFontOptions (PintaCore.Chrome.MainWindow.GetPangoContext (), options);
+		}
+		else
+		{
+			g.Antialias = Cairo.Antialias.None;
+			var options = new Cairo.FontOptions ();
+			options.Antialias = Antialias.None;
+			PangoCairo.Functions.ContextSetFontOptions (PintaCore.Chrome.MainWindow.GetPangoContext (), options);
```

Comment:
A minor refactoring would be to extract `options` and the `ContextSetFontOptions` call outside the `if` block to avoid the duplication in the two cases

Reply:
Done